lib/wahwah/mp3_tag.rb

```diff
@@ -52,6 +52,7 @@ def parse
     def parse_id3_tag
       @file_io.rewind
       signature = @file_io.read(6)
+      @file_io.rewind
```

Comment:
I noticed we were again reading the `file_io` but not potentially rewinding -- luckily I don't think it mattered too much as `ID3::V2#initialize`, and other related subclasses rewound things themselves, but I don't think it hurts to put it here as well.